### PR TITLE
Topic clearwork

### PR DESCRIPTION
**Pull request by CHenrie**
@jbest84 Please Review

### CODE REVIEW

File: src/Views/RelatedEditWidget.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        },
+        getEditSelect: function() {
+            var select = null;
+            if (this.formModel) {
+                select = this.formModel.getEditSelect();
+            }
+            return select;
+        },
+        onUpdateCompleted: function() {
+            if ((this.owner) && (this.owner._refreshClicked)) {
```
Comment (jbest84) [inline src/Views/RelatedEditWidget.js]: Unnecessary parens

**Comment by CHenrie** [comment]
@jbest84 Please Re Review after merge

File: src/Views/Opportunity/QuickEdit.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        enableStage: function(opportunityId) {
+            var promise, field, label;
+            field = this.fields['Stage'];
+            label = this.stageText;
+            if (!field) {
+                return;
+            }
+            field.disable();
+            promise = salesProcessUtility.getSalesProcessByEntityId(opportunityId);
+            if (promise) {
```
Comment (jbest84) [inline src/Views/Opportunity/QuickEdit.js]: getSalesProcessByEntityId always returns a promise object, which will always be truthy, no reason to check it.

File: src/Views/RelatedEditWidget.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                editView.sectionBeginTemplate = new Simplate([
+                     '<fieldset class="{%= ($.cls || $.options.cls) %}">'
+                ]);
+                editView.init();
+                editView._started = true;
+                editView.onUpdateCompleted = this.onUpdateCompleted.bind(this);
+            }
+            //Add the toolbar for save
+            toolBarNode = domConstruct.toDom(this.toolBarTemplate.apply(entry, this));
+            on(toolBarNode, 'click', lang.hitch(this, this.onInvokeToolBarAction));
```
Comment (jbest84) [inline src/Views/RelatedEditWidget.js]: Use Function.bind

**Comment by jbest84** [comment]
@CHenrie Commented on a couple of things, go ahead and merge once you resolve them.

**Pull request merged**